java/code/src/com/suse/manager/webui/utils/gson/PackageStateJson.java

```diff
@@ -89,6 +92,7 @@ public PackageStateJson(String nameIn, PackageEvr evrIn, String archIn) {
         this.arch = archIn;
         this.packageStateId = Optional.empty();
         this.versionConstraintId = Optional.empty();
+        this.packageEvr = evrIn;
```

Comment:
Why do we still store epoch, version and release as single values? Could we just change the getters to return packageEvr.getXXX() ?

Reply:
i reworked this again and removed the `packageEvr` again since i found a place that read `PackageStateJson` from json which would not have worked.